File: abc/abc184/b.cpp
// https://atcoder.jp/contests/abc184/submissions/27054572

#include <bits/stdc++.h>
using namespace std;
int main() {
  int n, x; cin >> n >> x;
  char c;
  while(cin >> c) {
    if(c == 'x') {if(x > 0) x--;} 
    else x++;
  }
  cout << x;
}

File: abc/abc184/b.java
// https://atcoder.jp/contests/abc184/submissions/43291060

import java.util.Scanner;

class Main {
  public static void main(String args[]) {
    Scanner stdIn = new Scanner(System.in);

    int n = stdIn.nextInt();
    int x = stdIn.nextInt();
    String s = stdIn.next();

    for(int i = 0; i < n; i++) {
      char c = s.charAt(i);

      if(c == 'o') {
        x++;
      } else {
        if(x > 0) {
          x--;
        }
      }
    }

    System.out.println(x);
  }
}
